finch/processes/wps_xsubsetpoint.py

```diff
@@ -108,7 +107,16 @@ def _subset_function(resource):
                 self.write_log(f"Subsetting file {count} of {n_files}", response, percentage)
 
             dataset = dataset[variables] if variables else dataset
-            return subset_gridpoint(dataset, lon=lon, lat=lat, start_date=start, end_date=end)
+
+            global_attributes = dataset.attrs
+            output_ds = None
+            for lon, lat in zip(longitudes, latitudes):
+                subset = subset_gridpoint(dataset, lon=lon, lat=lat, start_date=start, end_date=end)
+                subset = subset.expand_dims(["lat", "lon"])
+                output_ds = output_ds.combine_first(subset) if output_ds is not None else subset
```

Comment:
The different sites are ordered along which dimension ?

Reply:
Does it make more sense to have individual points with a multi-index coordinate of lon&lat together?
e.g. something like this:
```
index1 = pd.MultiIndex.from_arrays([longitudes, latitudes], names=['lon','lat'])
output = xr.Dataset(coords={'point_id': index1, 'time': subset.time}, attrs=subset.attrs)

```

@davidcaron I sent you a bit of code via email last week with an exemple not sure if you recieved?